portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -1294,6 +1294,19 @@ definition {
 		}
 	}
 
+	macro viewPredefinedValue {
```

Comment:
Hi @leilanyaragao , it seems that this macro can only be used for rich text, could you please rename it?